activemodel/test/cases/attributes_test.rb

```diff
@@ -24,6 +24,33 @@ class GrandchildModelForAttributesTest < ChildModelForAttributesTest
       attribute :string_field, default: "default string"
     end
 
+    class ModelWithGeneratedAttributeMethods
+      include ActiveModel::Attributes
+
+      attribute :foo
+    end
+
+    class ModelWithProxiedAttributeMethods
+      include ActiveModel::AttributeMethods
+
+      attribute_method_suffix '='
+
+      define_attribute_method(:foo)
+
+      def attribute=(_, _)
+      end
+    end
+
+    test "models that proxy attributes do not conflict with models with generated methods" do
```

Comment:
Not sure if the terminology is correct here 🤔

Reply:
fine by me.